index.js

```diff
@@ -0,0 +1,159 @@
+const express = require("express");
+const fs = require("fs");
+const app = express();
+
+app.use(express.json());
+
+app.get("/person/:name", (req, res) => {
+  const idadeRandom = Math.floor(Math.random() * (80 - 18) + 18);
+
+  const jsonResponse = JSON.parse(`{
+        "person": {
+            "name": "${req.params.name}",
+            "idade": "${idadeRandom}"
+        } 
+    }`);
+  res.send(jsonResponse);
+});
+
+let persons = [];
+
+(() => {
+  fs.readFile('./persons.txt', (err, data) => {
+    persons = JSON.parse(data)
+    if(err) {
+      console.log('Cannot read file')
+      console.log(err)
+    }
+  })
+})()
+
+function setPersons() {
+  fs.writeFile('./persons.txt', JSON.stringify(persons), 'utf-8', (err) => {
+    console.log(err)
+    })
+}
+
+app.post("/createperson", (req, res) => {
```

Comment:
Normally in REST style for creation of an entity we use POST /people, instead of the name of the operation.

Then you have GET /people/John, DELETE /people/John, GET /people (for listing all people).

Got it?

Reply:
in the next commit all the commands will be received in the same endpoint